ci_scripts/assert_news.py

```diff
@@ -99,12 +100,18 @@ def find_news_file(self):
         self.git.pull()
         master_branch_commit = self.git.get_current_commit()
         self.git.checkout(self.current_branch)
+        project_root = configuration.get_value(
+            ConfigurationVariable.PROJECT_ROOT)
+        news_dir_relative_path = os.path.relpath(news_dir,
+                                                 os.path.commonprefix(
+                                                     [news_dir, project_root]
+                                                 )
+                                                 )
```

Comment:
This section of code could do with a comment explaining what problem it is solving, it's not clear to me just be reading the code.

Although it is probably equivalent in this context I think `commonpath` would be what we would want, `commonprefix` is misleading if I understand the purpose of this section of code.

Lastly, would this be clearer if it was written with less indentation, e.g.

```
        news_dir_relative_path = os.path.relpath(
            news_dir, os.path.commonprefix([news_dir, project_root]))
```

Reply:
modifying this